docker/entrypoint.sh

```diff
@@ -1,18 +0,0 @@
-#!/bin/bash
-
-FIRST_BOOT_COMPLETE="first_boot_complete"
```

Comment:
Why the move from building on container build to building on starting the container? Are there any potential downsides we would need to worry about with regards to this? I imagine it might increase container size, or make it hard to use the container for local builds via mounting in a FF directory? Admittedly I'm not 100% sure what the current purpose/use-case of the docker file is, @jiazhihao might know more